addons/bus/controllers/main.py

```diff
@@ -12,6 +12,12 @@ def get_model_definitions(self, model_names_to_fetch, **kwargs):
             request.env['ir.model']._get_model_definitions(json.loads(model_names_to_fetch)),
         ))
 
+    @route("/bus/get_first_notification_dt", type="json", auth="public")
+    def get_first_notification_dt(self):
+        # sudo - bus.bus: getting oldest bus notification's create date is
+        # allowed to detect if the bus table was cleared during a disconnection.
+        return request.env["bus.bus"].sudo().search([], order="id ASC", limit=1).create_date
```

Comment:
This should use `search_read` with `fields=["create_date"]` to avoid fetching unnecessary fields.